lua/telescope/actions/init.lua

```diff
@@ -340,6 +340,23 @@ actions.git_create_branch = function(prompt_bufnr)
   end
 end
 
+--- Applies an existing git stash 
+---@param prompt_bufnr number: The prompt bufnr
```

Comment:
Make sure github actions for your fork are enabled so docgen is running. We are also running into a maximum amount of docs we can generate for that file (some bug in treesitter) so we are currently documenting new functions like this: 
```lua
-- TODO: add this function header back once the treesitter max-query bug is resolved
--
--
```
With two rather than 3 `-`. See 379